src/StackExchange.Redis/RedisDatabase.cs

```diff
@@ -2732,6 +2744,23 @@ public Task<RedisValue> StringSetRangeAsync(RedisKey key, long offset, RedisValu
             _ => throw new ArgumentException("Expiry time must be either Utc or Local", nameof(when)),
         };
 
+        private Message GetCopyMessage(in RedisKey source, RedisKey destination, int database, bool replace, CommandFlags flags)
+        {
+            if (database < 0) throw new ArgumentOutOfRangeException(nameof(database));
+            else if (database == 0)
```

Comment:
see previous comment about preferring negative to zero here, due to zero being a valid database number